test/integration/services/data/test-get-scenario.js

```diff
@@ -1,61 +0,0 @@
-const expect = require('chai').expect
```

Comment:
Why is this file being deleted? Looks like get-scenario isn't deleted in this PR

Reply:
I've reinstated it and moved the test only db queries into the test folder